### DISCUSSION TITLE
Is the storage.async_scorer parameter currently valid?

### DISCUSSION THREAD

**Discussion by yesletgo** [qdrant/qdrant#2469] [Q&A]
Is the storage.async_scorer parameter currently valid in qdrant 1.4.X ?

**Comment by generall** [comment]
yes. But it required kernel >5.4

**Reply by yesletgo** [reply]
kernel-lt-5.4.248-1.el7.elrepo.x86_64.rpm，is ok?

**Reply by generall** [reply]
no, it needs to be newer
